src/moveFile.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const moveFile = (oldPath, newPath) => {
+  const destPath = fs.existsSync(newPath)
+    ? path.join(newPath, path.basename(oldPath))
+    : newPath;
+
+  const isDestDir = destPath.endsWith('/');
```

Comment:
The check if the newPath is directory using endsWith('/') is not reliable. It should be replaced with fs.statSync(newPath).isDirectory()